cluster/ci/monitoring/mixins/prometheus_out/prometheus_rule_prow.yaml

```diff
@@ -260,3 +260,23 @@ spec:
       for: 5m
       labels:
         severity: slack
+  - name: sinker-missing
+    rules:
+    - alert: SinkerNotRemovingPods
+      annotations:
+        message: Sinker has not removed any Pods in the last hour, likely indicating
+          an outage in the service.
+      expr: |
+        absent(sum(rate(sinker_pods_removed[1h]))) == 1
+      for: 5m
+      labels:
+        severity: critical
```

Comment:
`critical` does not sound right: last time this happened everything worked for few weeks until we noticed we're using quite a lot of memory, but caused no real problems...

Reply:
Good catch.
Updated also the other places where we use `critical`.

We have not used this label `severity` yet and the meaning of the label follows the examples in the [prometheus-doc](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/#templating).

I think the `severity` here is the method to notify the Ops, here `slack` and `page` in the above doc.